src/components/Modal/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { modalPropsAtom } from '@/atoms/modalProps.atom'
+import { useRecoilValue } from 'recoil'
+import { Button } from '../Button'
+import * as S from './Modal.styles'
+
+const Modal = () => {
+  const { isOpen, message, onClose } = useRecoilValue(modalPropsAtom)
+  return (
+    <>
+      {isOpen && (
+        <S.Container>
+          <S.ContentCard>
+            <S.ModalBody>{message}</S.ModalBody>
+            <S.ModalFooter>
+              <Button
+                variant={'outlined'}
+                onClick={() => {
+                  onClose()
+                }}
```

Comment:
```suggestion
                onClick={onClose}
```

별건 아니지만 이렇게 간단히 쓰셔도 동일하게 동작합니다~

Reply:
- [refactor: 콜백함수 작성법 수정](https://github.com/2023-chambit-project/KOOSMOOS/pull/64/commits/d6f0d5ab2f53eb3057b811376b868638c2d53a49)
  - 맞네요.. 수정했습니다.!